elements/pfe-navigation/src/pfe-navigation--lightdom.scss

```diff
@@ -438,6 +433,32 @@ pfe-navigation-item {
 
             .pfe-navigation--column:nth-child(4) {
                 -ms-grid-column: 4;
+            }               
+
+            >*:nth-child(3),
+            >*:nth-child(4) {
+                margin-top: 0;
+            }
+        }
+
+        @media (min-width: #{pfe-breakpoint(xl)}) {
+            padding-bottom: 0;
+            display: -ms-grid;
+            grid-template-columns: repeat(4, 1fr);
+            -ms-grid-columns: 1fr 1fr 1fr 1fr;
+            align-items: center;
```

Comment:
@chrisdo1 I don't think you need this "align-items: center;" because you have it in @media (min-width: #{pfe-breakpoint(sm)})

Reply:
@alazzara removed